### PR TITLE
Wip/beol newton topics

### PR DESCRIPTION
**Pull request by SepidehAlassi**
- Adds new topics/subjects of "the newton project" as listnodes to subject_index
-  modifies labels of some of resources

### CODE REVIEW

File: webapi/_test_data/all_data/beol-data.ttl [hunk 1]
```diff
@@ -1459,3 +1459,40 @@
...
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Newton's Experiment of Glass" ;
+    knora-base:listNodePosition 0 ;
+    rdfs:label "Newton's Experiment of Glass"@en .
+
+<http://rdfh.ch/lists/0801/royalSocietyElection> a knora-base:ListNode ;
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Royal Society Election" ;
+    knora-base:listNodePosition 0 ;
+    rdfs:label "Royal Society Election"@en .
```
Comment (tobiasschweizer) [inline webapi/_test_data/all_data/beol-data.ttl]: Could you add a newline to the end of the file?

File: webapi/_test_data/all_data/beol-data.ttl [hunk 2]
```diff
@@ -1459,3 +1459,40 @@
 	knora-base:hasSubListNode <http://rdfh.ch/lists/0801/youth> , <http://rdfh.ch/lists/0801/personal_contact> , <http://rdfh.ch/lists/0801/correspondence> , <http://rdfh.ch/lists/0801/books> , <http://rdfh.ch/lists/0801/living_circumstances> , <http://rdfh.ch/lists/0801/miscellanea> ;
 	rdfs:label "Personal life"@en .
 
+<http://rdfh.ch/lists/0801/optics> a knora-base:ListNode ;
+    knora-base:hasRootNode <http://rdfh.ch/lists/0801/subject_index> ;
+    knora-base:listNodeName "Optics" ;
+    knora-base:listNodePosition 0 ;
```
Comment (tobiasschweizer) [inline webapi/_test_data/all_data/beol-data.ttl]: Could you check the `knora-base:listNodePosition`? It is set to 0 for all new nodes.

**Comment by SepidehAlassi** [comment]
> That looks good. Are you going to assign these to Newton letters? I assume your rationale is that one could find beol and Newton letters using the subject index, right?

yes, plan is that when one searches for a subject/topic will get all the corresponding  newton/beol (/hopefully soon Leibniz) letters

File: webapi/_test_data/all_data/beol-data.ttl (hunk 2 above)
Comment (SepidehAlassi) [reply webapi/_test_data/all_data/beol-data.ttl]: Thanks for bringing my attention to it. It is fixed now

File: webapi/_test_data/all_data/beol-data.ttl (hunk 1 above)
Comment (SepidehAlassi) [reply webapi/_test_data/all_data/beol-data.ttl]: done!

Comment (SepidehAlassi) [reply webapi/_test_data/all_data/beol-data.ttl]: done!

File: webapi/_test_data/ontologies/beol-onto.ttl [hunk 3]
```diff
@@ -1605,6 +1605,7 @@
                             owl:minCardinality "0"^^xsd:nonNegativeInteger ;
                             salsah-gui:guiOrder "1"^^xsd:nonNegativeInteger
                         ] ,
+
```
Comment (tobiasschweizer) [inline webapi/_test_data/ontologies/beol-onto.ttl]: You can get rid of this newline. I know that we are not consistent in formatting our Turtle ...

**Comment by tobiasschweizer** [comment]
> yes, plan is that when one searches for a subject/topic will get all the corresponding newton/beol (/hopefully soon Leibniz) letters

That's great!

**Pull request merged**